src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -207,8 +200,47 @@ public void run() {
     TypeModel type;
     if (baseIdentifier.equals(RESPONSE_PLACEHOLDER)) {
       view.variable(context.getNamer().getSampleResponseVarName(context));
+      boolean pageStreaming = context.getMethodConfig().getPageStreaming() != null;
+      boolean pageStreamingAndUseResourceName =
+          pageStreaming
+              && context
+                  .getFeatureConfig()
+                  .useResourceNameFormatOption(
+                      context.getMethodConfig().getPageStreaming().getResourcesFieldConfig());
+
+      // Compute the resource name format of output type and save that to local variables
+      if (pageStreamingAndUseResourceName) {
+        String typeName =
+            context
+                .getNamer()
+                .getAndSaveElementResourceTypeName(
+                    context.getTypeTable(),
+                    context.getMethodConfig().getPageStreaming().getResourcesFieldConfig());
+
+        Preconditions.checkArgument(
+            config.scan() == Scanner.EOF,
```

Comment:
We will want to support accessing members of a resource object, either directly with `.` or (most likely) indirectly via a `%` that will cause its rhs to resolve to the right accessor.

So include a TODO here for clarity, reminding us that the intent  is to allow this, but we have not yet gotten to this.